assets/js/components/settings/SettingsPlugin.js

```diff
@@ -65,6 +65,9 @@ export default function SettingsPlugin() {
 										checked={ showAdminBar }
 										onChange={ onAdminBarToggle }
 										disabled={ showAdminBar === undefined }
+										loading={
+											typeof showAdminBar !== 'boolean'
+										}
```

Comment:
Let's use the same condition here that we use for the `disabled` prop above as it's essentially the same logic.
```suggestion
										loading={ showAdminBar === undefined }
```